### PR TITLE
Bootstrap - De Uglify

### PR DESCRIPTION
**Pull request by medoix**
Started work on integrating Bootstrap so make it look nicer, what do you think? I can work on the other pages if you think this is an ok start.

### CODE REVIEW

**Comment by grant-olson** [comment]
This is awesome and will get merged.  A couple of things I'd like to discuss:
1. Previously there was a visual indication that a thread/channel/folder did or didn't contain new messages.  This seems to have disappeared.  Is it possible to bring back some visual indication?  I had added a class of .new and .old to the appropriate areas.
2. The send messages button http://localhost:4567/chans/ is too big now.  Can that be made less prominent or replaced with some sort of icon?  If not I might just remove the link from that page.
3. The menu was in three sections, and now it all flows together.  Is there any way to visually delimit the settings or break them out into a two-level menu structure?  (Write -> compose, Read -> Inbox, Chans, Lists, Settings -> Identities, settings)
4. Please add an entry to the HISTORY file, indicating that it was contributed by you, with either your handle or legal name, whatever you prefer.

Thanks!

**Comment by grant-olson** [comment]
Point 1. is the only thing a consider a blocker to merge.  Others are nice-to-have.

**Comment by grant-olson** [comment]
We'll also want new screenshots included in the README

File: views/layout.haml [hunk 1]
```diff
@@ -1,78 +1,100 @@
...
+!!!
+%html{:lang => "en"}
+  %head
+    %meta{:charset => "utf-8"}/
+    %title BMF - BitMessageForum
+    %meta{:content => "", :name => "description"}/
+    %meta{:content => "", :name => "author"}/
+    / Le HTML5 shim, for IE6-8 support of HTML elements
+    /[if lt IE 9]
+      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
```
Comment (grant-olson) [inline views/layout.haml]: Can this be included locally?  Someone is going to get paranoid about running third party code on their bitmessages.

File: views/layout.haml [hunk 2]
```diff
@@ -1,78 +1,100 @@
...
+      .body
+        padding: 40px 15px;
+        text-align: center;
+      }
+    %link{:href => "/css/bootstrap-responsive.css", :rel => "stylesheet"}/
+    / Le fav and touch icons
+    %link{:href => "images/favicon.ico", :rel => "shortcut icon"}/
+    %link{:href => "images/apple-touch-icon.png", :rel => "apple-touch-icon"}/
+    %link{:href => "images/apple-touch-icon-72x72.png", :rel => "apple-touch-icon", :sizes => "72x72"}/
+    %link{:href => "images/apple-touch-icon-114x114.png", :rel => "apple-touch-icon", :sizes => "114x114"}/
```
Comment (grant-olson) [inline views/layout.haml]: I don't think any of these images exist, do they?

**Comment by grant-olson** [comment]
Is there any reason we're not using the minified css/js files?

**Comment by medoix** [comment]
All things i will fix today, i was just wanting a "good" or "bad" idea from you :) I will add these changes to this pull in a couple of hours.

**Comment by medoix** [comment]
The layout is a template i use and i didnt want to do 100%  and then get a negative feedback, so now i know you like it i will bring everything local and fix up the rest.

**Comment by grant-olson** [comment]
I see you added a history entry.  Does that mean you're done or are more changes coming?

**Comment by medoix** [comment]
There will be more but i can always do another pull if you like, im actually not sure if the NEW indication has been fixed because i posted something but did not get the alert, i did not change anything with the actual logic so not sure why it would not display all of a sudden.

**Comment by grant-olson** [comment]
No I can wait.  I just didn't want to end up in a situation where you were waiting on me and I was waiting on you.  So give me an explicit heads up when you're ready.

Regarding the alerts, those will only show up every sixty seconds.  You should see a message in the console indicating "1 new messages..." then you should see it when you go to a page.

I saw one of the popup alerts, but...

The thing I was talking about that i don't see isn't the alert box, but the fact that the menu entries, the list of threads, and the list of messages should have a visual indication, so that (for example):

```
* The chans menu entry should look different than the settings menu entry.

* When you click through to chans, the new thread should have some indication that looks different from the old ones.

* When you click through the list of messages, the new message should have some other indicator.
```

But I'm starting to read up on Bootstrap and think I can figure it out if you cant.

Anyway, just let me know when you're ready and I'll do another round of reviews.

**Comment by grant-olson** [comment]
And you can more instantly force an alert by sending a message to yourself.  That happens more quickly than waiting on the new message thread.

**Comment by medoix** [comment]
That should fix what you are referring too, i also cleaned it up a little.

**Comment by medoix** [comment]
Can merge it in if you like and i can continue work on the main branch...

**Pull request merged**